packages/compiler/core/cli.ts

```diff
@@ -365,16 +361,21 @@ async function getCompilerOptions(
     }
   }
 
+  const pathArg = args["output-dir"] ?? args["output-path"] ?? config.outputDir ?? "./cadl-output";
+  const outputPath = resolvePath(process.cwd(), pathArg);
+  await mkdirp(outputPath);
+
   const cliOptions = resolveOptions(args);
+
   return {
     outputDir: outputPath,
     nostdlib: args["nostdlib"],
-    additionalImports: args["import"],
+    additionalImports: args["import"] ?? config["imports"],
```

Comment:
It feels like these should be able to combine between CLI and config file.

Reply:
But then should there be a way to clear the config. I'm not sure here.

Reply:
yeah everything in the config is override for now. Same as tsconfig, they did have an issue to add a way to append(when doing extends in particular) and it was rejected as it was adding too much complexcity(A little duplication sometimes isn't bad)